components/steering_wheel/include/Display/CommonDisplay.h

```diff
@@ -87,7 +87,7 @@ static void common_display(void)
 
 
     char tempMCU[10] = { 0U };
-    snprintf(tempMCU, 10, "% 2.*f", 2, IO.temp.mcu);
+    snprintf(tempMCU, 10, "% 2.*f", 2, IO_ANALOG.temp.mcu);
```

Comment:
Actually, now that I'm thinking about this more, maybe we should make a common `IO.h` file which contains a single `IO` struct with `analog` and `digital` structs as members... I feel like that will be less annoying to use than having to reference the structs separately

Reply:
no nevermind this is a bad idea